code/modules/vending/autodrobe.dm

```diff
@@ -15,35 +15,76 @@
 					/obj/item/clothing/under/rank/purpleclown = 1,
 					/obj/item/clothing/under/yogs/ronaldmcdonald = 1, // Yogs Change Added Ronald Costume
 					/obj/item/clothing/mask/yogs/ronald = 1, //Yogs Change Added Ronald Costume
+					/obj/item/clothing/mask/yogs/cluwne/happy_cluwne = 1, //yogs
+					/obj/item/clothing/mask/yogs/bananamask = 1, //yogs
+					/obj/item/storage/backpack/banana = 1, //yogs
+					/obj/item/clothing/mask/yogs/gigglesmask = 1, //yogs
+					/obj/item/storage/backpack/clownface = 1, //yogs
+					/obj/item/clothing/mask/yogs/scaryclown = 1, //yogs
+					/obj/item/clothing/under/yogs/scaryclown = 1, //yogs
+					/obj/item/clothing/shoes/clown_shoes/scaryclown = 1, //yogs
+					/obj/item/clothing/under/yogs/barber = 4, //yogs
+					/obj/item/clothing/head/yogs/boater = 4, //yogs
 					/obj/item/clothing/under/gladiator = 1,
 					/obj/item/clothing/head/helmet/gladiator = 1,
 					/obj/item/clothing/under/gimmick/rank/captain/suit = 1,
 					/obj/item/clothing/head/flatcap = 1,
 					/obj/item/clothing/suit/toggle/labcoat/mad = 1,
-					/obj/item/clothing/shoes/jackboots = 1,
+					/obj/item/clothing/shoes/jackboots = 10, //yogs
+					/obj/item/clothing/under/yogs/bluecoatuniform = 5, //yogs
+					/obj/item/clothing/suit/yogs/bluecoatcoat = 5, //yogs
+					/obj/item/clothing/head/yogs/tricornhat = 5, //yogs
+					/obj/item/clothing/head/yogs/microwave = 1, //yogs
+					/obj/item/clothing/head/yogs/drinking_hat = 1, //yogs
+					/obj/item/clothing/suit/yogs/beaker = 1, //yogs
+					/obj/item/clothing/suit/yogs/facebook = 1, //yogs
+					/obj/item/clothing/under/yogs/persskinsuit = 3, //yogs
 					/obj/item/clothing/under/schoolgirl = 1,
 					/obj/item/clothing/under/schoolgirl/red = 1,
 					/obj/item/clothing/under/schoolgirl/green = 1,
 					/obj/item/clothing/under/schoolgirl/orange = 1,
+					/obj/item/clothing/suit/yogs/gothic = 1, //yogs
 					/obj/item/clothing/head/kitty = 1,
 					/obj/item/clothing/under/skirt/black = 1,
+					/obj/item/clothing/under/yogs/zootsuit = 1, //yogs
+					/obj/item/clothing/head/yogs/zoothat = 1, //yogs
+					/obj/item/clothing/under/yogs/hamiltonuniform = 1, //yogs
+					/obj/item/clothing/suit/yogs/hamiltoncoat = 1, //yogs
 					/obj/item/clothing/head/beret = 1,
 					/obj/item/clothing/accessory/waistcoat = 1,
 					/obj/item/clothing/under/suit_jacket = 1,
+					/obj/item/clothing/suit/hooded/sandsuit = 1, //yogs
```

Comment:
There are a *lot* of patches of ``//yogs`` lines in this file.

Do these need to be in a particular order? Can you not just put them all together and just have a nice, big ``//yogs start //yogs end`` block? It'll help reduce the shittiness of dealing with any merge conflicts.

An even sexier alternative would be to take these all out and, in ``/obj/machinery/vending/autodrobe/Initialize()``, do, like,
```dm
/obj/machinery/vending/autodrobe/Initialize()
	..()
	products += list(
		obj/fuckingthingA = 1,
		obj/fuckingthingB = 1,
		/* Et cetera, et cetera... */
	)
```
Assuming Initialize for this thing is not already defined by /tg/, which I don't think it is.